docs/content/guides/getting-started/grid-size.md

```diff
@@ -31,21 +31,37 @@ Handsontable doesn't observe CSS changes for containers out of the box.
 If you'd like to observe it, you can define the dimensions in the configuration object or create your own observer.
 :::
 
-### Pass the size in the configuration object
+### Pass the size in the configuration
 
+You can pass width and height values to Handsontable as numbers or possible CSS values for the "width"/"height" properties:
+::: only-for javascript
 ```js
 {
-    width: '100px',
-    width: '75%',
-    width: 100 // For a better compatibility we convert number into pixels
+  width: '100px',
+  height: '100px',
+}
+or
+{
+  width: '75%',
+  height: '75%',
 }
-...
+or 
 {
-    height: '100px',
-    height: '75%',
-    height: 100 // For a better compatibility we convert number into pixels
+  width: 100,
+  height: 100,
 }
 ```
+:::
+
+::: only-for react
+```jsx
+  <HotTable height={100} width={100} />
+or
+  <HotTable height="75%" width="75%" />
+or
+  <HotTable height="100px" width="100px" /> 
+```
+:::
```

Comment:
This is a change for good, but could you please put the `or` words outside of the snippets? It does not compile in my head.

Reply:
changed